sdk/eventhub/event-hubs/rollup.base.config.js

```diff
@@ -96,13 +98,40 @@ export function browserConfig(test = false) {
           }
         }
       ),
+
+      shim({
```

Comment:
The counterpart to this in service bus has a helpful comment just before we shim. Can we add the same here as the reasons to do this is the same as service bus?